src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_vr_language_change_notification.cc

```diff
@@ -72,12 +72,13 @@ void OnVRLanguageChangeNotification::Run() {
   (*message_)[strings::params][strings::function_id] =
       static_cast<int32_t>(mobile_apis::FunctionID::OnLanguageChangeID);
 
-  const ApplicationSet& accessor =
-      application_manager_.applications().GetData();
+  ApplicationSet applications;
+  {
+    DataAccessor<ApplicationSet> accessor = application_manager_.applications();
+    applications = accessor.GetData();
+  }
```

Comment:
Agreed, I don't see any way that the applications lock could still be held in the original code. The DataAccessor does not a have a scope, so the lock would release immediately after `GetData()`. It seems more like a race condition fix since we are copying the application set rather than using a const reference to it.

Reply:
Agreed, fixed. 
fdd4b3000e